assets/src/styles/header.scss

```diff
@@ -0,0 +1,48 @@
+#skip a {
```

Comment:
As an inclusion file / partial, I think this file should be named `_header.scss`